libr/bin/p/bin_elf.inc

```diff
@@ -984,7 +984,7 @@ static RList* patch_relocs(RBin *b) {
 		return NULL;
 	}
 	if (!io->cached) {
-		eprintf ("Warning: run r2 with -e bin.cache=true to fix relocations in disassembly\n");
+		R_LOG_WARN ("Run r2 with -e bin.cache=true to fix relocations in disassembly");
```

Comment:
```suggestion
		eprintf ("Run r2 with -e bin.cache=true to fix relocations in disassembly\n");
```

Reply:
keep it as eprintf() as this is a temporal warning that we want to be always displayed until we get the right relocs implementation in rio.